cli.js

```diff
@@ -90,6 +92,31 @@ const save = () => {
 
     // TODO handle creating files
     // TODO handle deleting files
+    const oldKeys = Object.keys(previousDocument);
+    const newKeys = Object.keys(currentDocument);
+    const removedKeys = oldKeys.filter((key) => !newKeys.includes(key));
+    removedKeys.forEach((key) => {
+      fs.unlinkSync(fullPath + '/' + previousDocument[key].name, (err) => {
```

Comment:
Interesting that this uses `fullPath`, but the `writeFileSync` call worked fine without this.

Maybe it's not required?

```suggestion
      fs.unlinkSync(previousDocument[key].name, (err) => {
```